src/Solnet.KeyStore/Crypto/RandomBytesGenerator.cs

```diff
@@ -18,9 +19,9 @@ public byte[] GenerateRandomSalt()
 
         private static byte[] GenerateRandomBytes(int size)
         {
-            var bytes = new byte[size];
+            Span<byte> bytes = stackalloc byte[size];
             Random.NextBytes(bytes);
-            return bytes;
+            return bytes.ToArray();

```

Comment:
Same thing here, the array is returned, so a new copy is created in the heap. Revert to prevent more CPU usage in the copy and pointless stack allocation.